tracer/test/test-applications/integrations/Samples.InstrumentedTests/Vulnerabilities/InstrumentationTestsBaseExcluded.cs

```diff
@@ -0,0 +1,20 @@
+// <copyright file="InstrumentationTestsBaseExcluded.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System.Reflection;
+
+namespace DataDog.Excluded.Samples.InstrumentedTests.Iast.Vulnerabilities;
```

Comment:
Nice :)

Reply:
I forgot to revert that part. It's not needed anymore as we don't choose to look for `.Invoke()` 🙈